### PR TITLE
Fixed jewelery option level multipliers for max Ability and Mana increase

### PR DESCRIPTION
**Pull request by ze-dom**
Calculations of Max Ability and Mana increase for jewelery was multiplying by values 1.01/2.02/3.03/4.04 (respectively for +1%/2%/3%/4% level options). Evidence:
[Screencast from 12-09-2024 11:03:33.webm](https://github.com/user-attachments/assets/89eb865d-5b9a-44fe-b342-f5d3abab0900)
[Screencast from 12-09-2024 11:17:22.webm](https://github.com/user-attachments/assets/5ba49e6c-2022-4932-a28e-6e0262ed82d8)

After fix:
[Screencast from 13-09-2024 21:09:03.webm](https://github.com/user-attachments/assets/87ab967b-6995-4044-a8c0-66fb88ef0b08)

### CODE REVIEW

**Comment by sd3420625** [comment]
In my opinion, this is a relatively useful and benign bug, which solves the problem of agile characters consuming mana too quickly :)

**Comment by sven-n** [comment]
Sorry for the late reply 😅
The fix makes perfectly sense, thank you :)
Just one thing needs to be done as well: A configuration update plugin to fix existing configurations. See `MUnique.OpenMU.Persistence.Initialization.Updates`. We need to fix this at least for the season 6 config. To add a new update plugin:
* Add a new enum value in `UpdateVersion`.
* Implement a new plugin by inheriting from `UpdatePlugInBase`.

Can you add that please?

File: src/Persistence/Initialization/Updates/FixMaxManaAndAbilityJewelryOptionsUpdateSeason6.cs [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    public override bool IsMandatory => true;
+
+    /// <inheritdoc />
+    public override DateTime CreatedAt => new(2024, 09, 26, 10, 00, 0, DateTimeKind.Utc);
+
+    /// <inheritdoc />
+    protected override async ValueTask ApplyAsync(IContext context, GameConfiguration gameConfiguration)
+    {
+        var itemOptions = gameConfiguration.ItemOptions
+            .Where(io => io.Name == "Jewelery option Maximum Mana" || io.Name == "Jewelery option Maximum Ability");
```
[the remark this answers is not in the document]
Comment (sven-n) [reply src/Persistence/Initialization/Updates/FixMaxManaAndAbilityJewelryOptionsUpdateSeason6.cs:52]: You can assume that the Guids are constant when they don't look like random.

**Comment by ze-dom** [comment]
@sven-n , all set, I think :)

**Comment by sven-n** [comment]
Thank you :)

**Pull request merged**